app/js/lib/view_handler/fee_option_switcher.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for enabling and disabling elements if the update value exceeds a certain threshold
+	 * @class
+	 */
+	FeeOptionSwitcher = {
+		isDisabled: null, // Null to always trigger show/hide on first call to update
+		thresholdValue: 0,
+
+		update: function ( value ) {
+			if ( value > this.thresholdValue ) {
+				if ( this.isDisabled === true ) {
+					return;
+				}
+				this.toggler.disableElement();
+				this.isDisabled = true;
+			} else {
+				if ( this.isDisabled === false ) {
+					return;
+				}
+				this.toggler.enableElement();
+				this.isDisabled = false;
+			}
+		}
+	},
+
+	ElementAvailabilityToggler = {
+		element: null,
```

Comment:
An abstract class plus an implementing class are not strictly necessary. You probably got this idea from the visibility switcher where we have different methods of switching the visibility (animated and non-animated). Since the availability is always toggled the same way, you could put the code from `SimpleElementAvailabilityToggler` directly into `FeeOptionSwitcher`